corehq/middleware.py

```diff
@@ -220,37 +215,7 @@ def process_view(self, request, view_func, view_args, view_kwargs):
                 scope.set_tag('domain', request.domain)
 
 
-session_logger = logging.getLogger('session_access_log')
-
-
-def log_call(func):
-    def with_logging(*args, **kwargs):
-        session_logger.info("\n\n\n")
-        session_logger.info(func.__name__ + " was called")
-        session_logger.info("\n\n\n")
-        for line in traceback.format_stack():
-            white_list = ['corehq/', 'custom/', func.__name__]
-            if any([c in line for c in white_list]):
-                session_logger.info(line.strip())
-        return func(*args, **kwargs)
-    return with_logging
-
-
-def decorate_all_methods(decorator):
-    def decorate(cls):
-        for attr in dir(cls):
-            if "__" not in attr and callable(getattr(cls, attr)):
-                setattr(cls, attr, decorator(getattr(cls, attr)))
-        return cls
-    return decorate
-
-
-@decorate_all_methods(log_call)
-class LoggingSessionStore(SessionStore):
-    pass
-
-
-class LoggingSessionMiddleware(SessionMiddleware):
+class BypassSessionMiddleware(SessionMiddleware):
```

Comment:
nit: how about `SelectiveSessionMiddleware` - I think this is better since the currently name implies it will always bypass the session.

Reply:
Sure